README.md

```diff
@@ -8,9 +8,20 @@ Julia simulator for [Functional Mockup Interface (FMI)](https://fmi-standard.org
 ## Getting started
 
 ### Installing JuliaFMI
-```
+```julia
+julia> import Pkg
 julia> Pkg.clone("https://github.com/AnHeuermann/JuliaFMI.git")
 ```
+You can force rebuilding with
+```julia
+(v1.2) pkg> build JuliaFMI
+```
+
+### Run the tests
+```julia
+(v1.2) pkg> test JuliaFMIt
```

Comment:
I guess the additional `t` is a typo, i.e. 
```suggestion
(v1.2) pkg> test JuliaFMI
```

Reply:
Yes, thanks for finding the typo